webhooks.py

```diff
@@ -163,7 +171,7 @@ def index():
     for s in scripts:
 
         proc = Popen(
-            [s, tmpfile, event],
+            [s, tmpfile, event, str(branch)],
             stdout=PIPE, stderr=PIPE
```

Comment:
I don't have good idea how to refactor this. I just found annoying and hard to get the branch name from inside a shell script - and yes I am managing an auto builder of project branches with single hook script.